shark/endpoints/testing.py

```diff
@@ -1,12 +1,51 @@
-from fastapi import APIRouter, Request
+from fastapi import APIRouter, Request, Depends
+
+from sqlalchemy.orm import Session
+
+from shark.utils.crud import get_db
+from shark.utils import models, schemas
+from shark.utils.database import engine
```

Comment:
```suggestion
from .utils import models, schemas
from .utils.crud import get_db
from .utils.database import engine
```

These should be relative imports.

`utils` dif should also have an `__init__` importing relative functions/classes